lib/ansible/modules/cloud/ovirt/ovirt_host.py

```diff
@@ -344,13 +348,25 @@ def post_reinstall(self, host):
             timeout=self.param('timeout'),
         )
 
+    def raise_host_exception(self):
+        events = self._connection.system_service().events_service().list(max=30)
```

Comment:
I've meant that you should call `self._connection.system_service().events_service().list(max=1)` at the begging of the module, and then list the events only from that specific event ID like `search="from_=%s" % event.id`. You can check `wait_condition` in `check_for_upgrade` for an example.